examples/examples-by-storage/main.py

```diff
@@ -8,14 +8,20 @@
 import demos
 from model import train
 from modelstores import create_model_store
+from modelstore.utils.exceptions import ModelDeletedException, ModelNotFoundException
```

Comment:
I've added two custom exceptions to `modelstore` (see below) - to differentiate between models that never existed vs. models that have been deleted. There's some scope to extend these a bit; for example, `modelstore` doesn't currently have a `DomainNotFoundException`. One for the future